c10/cuda/CUDACachingAllocator.h

```diff
@@ -90,6 +90,9 @@ struct DeviceStats {
   // COUNT: total number of oversize blocks requiring malloc
   Stat oversize_segments;
 
+  // SUM: bytes requested by client code
```

Comment:
Is there a reason this wouldn't mirror `allocated_bytes` in recording min/max etc?

Reply:
I've now implemented this like the rest of the stats e.g. `allocated_bytes`, `reserved_bytes`